service/src/main/kotlin/fi/espoo/evaka/pis/ParentshipQueries.kt

```diff
@@ -56,12 +58,15 @@ fun Handle.getParentships(
         JOIN person head ON fc.head_of_child = head.id
         WHERE (:headOfChild::uuid IS NULL OR head_of_child = :headOfChild)
         AND (:child::uuid IS NULL OR child_id = :child)
+        AND daterange(fc.start_date, fc.end_date, '[]') && daterange(:from, :to, '[]')
```

Comment:
Pro tip: you can bind period objects directly as parameters, so you can do `.bindNullable("period", period")` and use `&& :period` in the query

Reply:
Yeah, but `Period` doesn't support null start dates and I'm a bit afraid to change it 😅